testing/trino-testing/src/main/java/io/trino/testing/containers/HiveHadoop.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.testing.containers;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.net.HostAndPort;
+import org.testcontainers.containers.Network;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+public class HiveHadoop
+        extends BaseTestContainer
+{
+    private static final String RUN_CMD = "/usr/local/hadoop-run.sh";
+
+    public static final String DEFAULT_IMAGE = "ghcr.io/trinodb/testing/hdp2.6-hive:41";
```

Comment:
Is it possible to take the image version from the product tests (or some static place) to make sure product tests and integration tests use consistent images? Or is it intentionally kept separate?

Reply:
Well we could create smth simillar to EnvironmentDefaults from trino-product-test-launcher in trino-testing-services.
E.g DockerImagesDefaults. 
Then point EnvironmentDefaults to it as well as test in hive.
WDYT ?

Reply:
That sounds good. cc: @findepi @losipiuk WDYT?

Reply:
@findepi @losipiuk @hashhar I will implement it as last commit. If we don't like it I will drop it and leave it as it is.